pallets/order-book/src/tests/order_book.rs

```diff
@@ -532,25 +573,37 @@ fn should_not_place_limit_order_that_doesnt_meet_restrictions_for_orders_in_pric
 
             buy_order.id += 1;
             buy_order.owner = account.clone();
+            // should ideally be set through `LimitOrder::new`
+            // but let's do it in a hacky way for simplicity
+            buy_order.expires_at += 1;
             sell_order.id += 1;
             sell_order.owner = account;
+            // should ideally be set through `LimitOrder::new`
+            // but let's do it in a hacky way for simplicity
```

Comment:
Comments are unnecessary

Reply:
I intended to place them so that people that see this test don't use the same approach where it's not appropriate